pnpm-workspace.yaml

```
@@ -0,0 +1,5 @@
+packages:
+  - '.'
+  - 'web'
+  # Root directory for shared scripts and dependencies
+  # The api package uses Go and doesn't need to be included in the workspace
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)